multisig/proposals/arbitrumsepolia/vip-001/index.ts

```diff
@@ -0,0 +1,181 @@
+      params: [arbitrumsepolia.RESILIENT_ORACLE, "setOracle(address,address,uint8)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.RESILIENT_ORACLE, "enableOracle(address,uint8,bool)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.RESILIENT_ORACLE, "setTokenConfig(TokenConfig)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.CHAINLINK_ORACLE, "setTokenConfig(TokenConfig)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.CHAINLINK_ORACLE, "setDirectPrice(address,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [BOUND_VALIDATOR, "setValidateConfig(ValidateConfig)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: BOUND_VALIDATOR,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_WBTC, CHAINLINK_BTC_FEED, 144000]],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          MOCK_WBTC,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_WETH, CHAINLINK_ETH_FEED, 144000]],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          MOCK_WETH,
+          [
+            arbitrumsepolia.CHAINLINK_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_USDC, CHAINLINK_USDC_FEED, 144000]],
```

Comment:
We are usually setting a stale period of 26h (93600 seconds) for feeds with a heartbeat of 24h, like this one

Reply:
Done https://github.com/VenusProtocol/vips/pull/247/commits/c573c14b8980fedd4ab11d46d064198f238595b3